quilt/README.md

```diff
@@ -0,0 +1,241 @@
+execute `quilt get github.com/amplab/ray`. This will clone the ray repository
+into your `QUILT_PATH`, and will also download imports of the specs that were
+downloaded.
+
+## Step 2: Configure `ray-runner.spec` & Cloud Provider
+In [`ray-runner.spec`](ray-runner.spec), there are two things that will have to
+be filled in.
+
+### Configure SSH Authentication
+Quilt-managed Machines use public key authentication to control SSH access.
+To start using the Ray cluster, we will need to access the Master VM.
+
+If you would like to use `githubKey` authentication, go to line 24 of 
+[`ray-runner.spec`](ray-runner.spec) and fill in  `(define githubKey
+"YOUR_GITHUB_USERNAME")` appropriately. Instructions on adding a key to your
+GitHub account can be found
+[here](https://help.github.com/articles/generating-an-ssh-key/).
+
+If you would ike to use an SSH key instead, replace `(define githubKey
+"YOUR_GITHUB_USERNAME")` with `(define sshkey "YOUR_PUBLIC_KEY")`.
+
+`"YOUR_PUBLIC_KEY"` should be a string with the contents of your
+`~/.ssh/id_rsa.pub`, e.g.: `(define sshkey "ssh-rsa
+AAAAB3NzaC1yc2EAAAADAQABAAABAQ example@example.com")`.
+
+### Choose a Namespace
+Running two Quilt instances with the same Namespace is not supported.
+If you are sharing a computing cluster with others, it would be a good idea to
+change `(define Namespace "CHANGE_ME")` on line 11 to a different name.
+
+### Configure a Cloud Provider
+Below we discuss how to setup Quilt for Amazon EC2. Other providers are
+supported as well, including Microsoft Azure and Google Compute Engine. 
+Since Quilt deploys systems consistently across providers, the details of the
+rest of this document will apply no matter what provider you choose.
+
+For Amazon EC2, you'll first need to create an account with [Amazon Web
+Services](https://aws.amazon.com/ec2/) and then find your
+[access credentials](http://docs.aws.amazon.com/cli/latest/userguide/cli-chap-getting-set-up.html#cli-signup).
+That done, you simply need to populate the file `~/.aws/credentials`, with your
+Amazon credentials:
+
+```
+[default]
+aws_access_key_id = <YOUR_ID>
+aws_secret_access_key = <YOUR_SECRET_KEY>
+```
+
+## Step 3: Run `ray-runner.spec`
+
+### Start the Quilt daemon
+Execute `quilt daemon` in one terminal session. This will start the quilt
+process (and hang). Do not close this session. Open a new terminal session
+(<kbd>Ctrl</kbd> + <kbd>T</kbd> or <kbd>&#8984;</kbd> + <kbd>T</kbd> on OS X).
+
+### Run `ray-runner.spec`
+We suggest you read through [`ray.spec`](ray.spec) and
+[`ray-runner.spec`](ray-runner.spec) to get a feel of how the infrastructure is
+defined.
+
+While in the `path/to/amplab/ray/quilt/` directory, execute `quilt run
+ray-runner.spec`. Quilt will set up `nWorker + 1` VMs. You will see logs pop up
+on the terminal session where you executed `quilt daemon`.
+
+### Accessing the VM
+It will take a while for the VMs to boot up, for Quilt to configure the network,
```

Comment:
I would rewrite this section in terms of the `quilt machines` command instead of making them look through the logs.  You might even suggest they `watch quilt machines` until everything says connected.